constructor/coordinator/coordinator.go

```diff
@@ -140,7 +154,7 @@ func (c *Coordinator) findJob(
 
 	// Check if ErrCreateAccount, then create account if exists
```

Comment:
nit: by "if exists" do you mean if the account exists or if the error exists?

Reply:
🤦

Reply:
This is a remnant of an old change I made